lib/page/needed.rb

```diff
@@ -0,0 +1,28 @@
+module Page
+  class Needed
+    def initialize(access_token = '')
+      @access_token = access_token
+      warn 'No GITHUB_ACCESS_TOKEN found' if @access_token.to_s.empty?
+    end
+
+    def to_find
+      client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Find'
+    end
+
+    def to_scrape
+      client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Scrape'
+    end
+
+    def to_finish
+      client.issues 'everypolitician/everypolitician-data', labels: 'New Country,3 - WIP'
+    end
+
+    private
+
+    def client
+      client = Octokit::Client.new(access_token: @access_token)
```

Comment:
This is a slight change in behaviour from the original. Previously if `access_token` was empty, the call would be a plain `Octokit::Client.new`, whereas you're now calling `Octokit::Client.new(access_token: '')`

That _seems_ to Do The Right Thing, but it's not obvious from the Octokit docs that it's guaranteed to. 

However, running this from here _without_ an access token currently fails with "403 - API rate limit exceeded" — which we're not capturing.

As in production we always provide an access token anyway, I suspect it might make things much simpler if we just declare that it's compulsory rather than optional…